fairlearn/metrics/_metrics_engine.py

```diff
@@ -3,9 +3,17 @@
 
 import numpy as np
 
-from ._group_metric_result import GroupMetricResult
 from ._input_manipulations import _convert_to_ndarray_and_squeeze
 
+_OVERALL = 'overall'
+_BY_GROUP_FORMAT = 'group_{0}'
+_MIN = 'min'
+_MAX = 'max'
+_RANGE = 'range'
+_RANGE_RATIO = 'range_ratio'
+_ARGMIN = 'argmin'
+_ARGMAX = 'argmax'
```

Comment:
I personally find this less readable than having those literals directly in the code. Especially when they're used pretty much only once.